apps/ledger-live-mobile/src/families/ripple/ScreenEditTag.tsx

```diff
@@ -65,6 +58,7 @@ function RippleEditTag({ route, navigation }: Props) {
   const onValidateText = useCallback(() => {
     if (!account) return;
     const bridge = getAccountBridge(account);
+    // @ts-expect-error FIXME: no current / next navigation param?
```

Comment:
Curious: do you know what's the error to fix here? I'm seeing a lot of `navigate` in the code that don't provide currentNavigation / nextNavigation params?